libs/vscode/nx-cloud-view/src/lib/nx-cloud-service/nx-cloud-service.ts

```diff
@@ -459,27 +460,22 @@ export class NxCloudService extends StateBaseService<InternalState> {
   }
 
   private async loadAndSetIsUsingCloudRunner(reset = false) {
-    const isConnected = await this.isUsingCloudRunner(reset);
+    const isConnected = Boolean(await getNxCloudRunnerOptions(reset));
     this.setState({ isUsingCloudRunner: isConnected });
   }
 
   private async loadAndSetIsPrivateCloud() {
     const nxWorkspaceConfig = (await getNxWorkspace()).workspace;
-    const cloudRunnerUrl =
-      nxWorkspaceConfig.tasksRunnerOptions?.default?.runner ===
-        '@nrwl/nx-cloud' ||
-      nxWorkspaceConfig.tasksRunnerOptions?.default?.runner == 'nx-cloud'
-        ? nxWorkspaceConfig.tasksRunnerOptions?.default?.options?.url
-        : undefined;
-
-    this.setState({ cloudRunnerUrl });
+    const cloudRunnerOptions = await getNxCloudRunnerOptions();
+
+    this.setState({ cloudRunnerUrl: cloudRunnerOptions?.url });
   }
 
   private async generateListOfFirstCommands(): Promise<void> {
     const generateAndSetFirstCommands = async () => {
       const nxConfig = (await getNxWorkspace()).workspace;
-      const cacheableOperations =
-        nxConfig.tasksRunnerOptions?.default?.options.cacheableOperations;
+      const cacheableOperations = await getCacheableOperations();
```

Comment:
What do we use this for? We are working towards `cache` being settable on a per target basis, so this may not be accurate soon.

Reply:
This is used to generate some suggestions of commands the user could run to see task runs in nx cloud. I ultimately get the info from going over all targets in the project graph so this should remain accurate even then.